ts/src/pro/bitfinex2.ts

```diff
@@ -60,7 +60,7 @@ export default class bitfinex2 extends bitfinex2Rest {
         const checksum = this.safeValue (this.options, 'checksum', true);
         if (checksum && !client.subscriptions[messageHash]['checksum'] && (channel === 'book')) {
             client.subscriptions[messageHash]['checksum'] = true;
-            await client.send ({
+            client.send ({
```

Comment:
@pcriadoperez hum I think we need this `await` here otherwise we get
`/Users/cjg/Git/ccxt2/ccxt/python/ccxt/pro/bitfinex2.py:66: RuntimeWarning: coroutine 'ClientWebSocketResponse.send_str' was never awaited`

because 
```
    def send(self, message):
        if self.verbose:
            self.log(iso8601(milliseconds()), 'sending', message)
        return self.connection.send_str(message if isinstance(message, str) else json.dumps(message, separators=(',', ':')))
```
⬆️  send_str is async